### PR TITLE
Add link to docs on README

### PR DESCRIPTION
**Pull request by maurogeorge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,6 +347,13 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+## Documentation
+
+AMS has two main sets of documentation: the
+[Docs](https://github.com/rails-api/active_model_serializers/tree/master/docs),
+which is a comprehensive guide and in
+[RDoc](http://www.rubydoc.info/github/rails-api/active_model_serializers) format.
```
Comment (bf4) [inline README.md]: Thanks for this, :+1: 

should be above the `release  candidate plz read`. Also, I don't know if we can link to different versions of docs, or just note these are for master.

**Comment by beauby** [comment]
:+1: Let's add docs to `0.8` and `0.9` as well, just so that people are not confused.

**Comment by bf4** [comment]
@beauby or master just has docs for different versions because..

**Comment by beauby** [comment]
@bf4 Sorry, I didn't get what you meant.

**Comment by bf4** [comment]
No misunderstanding just thinking aloud... Might be baad idea

B mobile phone

**Comment by NullVoxPopuli** [comment]
I think the readme on master should only have docs for master, and maybe docs for upgrading from previous versions to master. :-\

**Comment by maurogeorge** [comment]
Hi guys, I updated the code move the docs to above the `release candidate plz read`.

I agree with the @NullVoxPopuli I think the master should only have docs for master, and maybe docs for upgrading from previous versions to master.

Let me know if this is ok.
Thanks :heart:

**Comment by bf4** [comment]
Closed by e4342aa

**Comment by maurogeorge** [comment]
Thanks @bf4

**Pull request closed without merging**